lessons/bn/basics/basics.md

```diff
@@ -22,15 +22,17 @@
 
 ### ইন্টারএ্যাক্টিভ মোড
 
-এলিক্সিরের ইন্টারএ্যাক্টিভ শেল এর নাম `iex` - এই ইন্টারএ্যাক্টিভ প্রম্পট এ আমরা এলিক্সির এক্সপ্রেশন টাইপ করে সাথে সাথে আউটপুট পেতে পারি । 
+এলিক্সিরের ইন্টারএ্যাক্টিভ শেল এর নাম `iex` - এই ইন্টারএ্যাক্টিভ প্রম্পট এ আমরা এলিক্সির এক্সপ্রেশন টাইপ করে সাথে সাথে আউটপুট পেতে পারি ।
 
-ইন্টারএ্যাক্টিভ শেল চালু করত টাইপ করুন - `iex`:
+ইন্টারএ্যাক্টিভ শেল চালু করতে কমান্ডলাইনে টাইপ করুন - `iex`:
 
     Erlang/OTP {{ site.erlang.OTP }} [erts-{{ site.erlang.erts }}] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
 
     Interactive Elixir ({{ site.elixir.version }}) - press Ctrl+C to exit (type h() ENTER for help)
     iex>
 
+বি:দ্র: উন্ডোজপাওয়ার শেলে আপনাকে `iex.bat` টাইপ করতে হবে।
+
 আসুন আমরা কিছু এক্সপ্রেশন টাইপ করি এবং সেগুলোর আউটপুট দেখে নেই :
```

Comment:
The symbol `:` can be replaced with the Bengali version `ঃ`
Example: `নেই :` should be `নেইঃ`

Reply:
@wasi0013 bhai yes, traditionally বিঃদ্রঃ is written, so I think we should go that route. waiting for your confirmation.